market_maker_keeper/cex_api.py

```diff
@@ -36,25 +36,28 @@ class CEXKeeperAPI:
     Define a common abstract API for keepers on centralized and hybrid exchanges
     """
 
-    def __init__(self, arguments: Namespace, pyex_api: PyexAPI):
+    def __init__(self, arguments: Namespace, pyex_api: PyexAPI, standard_pyex: bool):
```

Comment:
The `standard_pyex` variable tastes a little off.  Does it mean `uses_order_history_reporter_and_order_book_manager`?  Are those two facilities inseparable?  Should we have separate methods for `init_history_reporter` and `init_order_book_manager` and let subclasses override behavior where needed?

Reply:
Good point. Creating an `init_order_book_manager` method that could be subclassed would be cleaner.

Reply:
I continued passing arguments and `pyex_api` for the typechecking in init, even though they weren't strictly necessary